R/build-articles.R

```diff
@@ -55,7 +55,7 @@ build_articles <- function(pkg = ".", path = "docs/articles", depth = 1L, encodi
   on.exit(set_pkgdown_env(old))
 
   pkg <- as_pkgdown(pkg)
-  path <- rel_path(path, pkg$path)
+  path <- rel_path(path)
```

Comment:
This is fundamentally changing where pkgdown builds the website, and I think it's a bad idea to change it.